cmd/kapacitor/main.go

```diff
@@ -2246,3 +2258,48 @@ func doBackup(args []string) error {
 	}
 	return nil
 }
+
+func watchUsage() {
+	var u = `Usage: kapacitor watch <output file>
+
+	Watch logs associated with a task.
+`
+	fmt.Fprintln(os.Stderr, u)
+}
+
+func doWatch(args []string) error {
+	if len(args) != 1 {
+		return errors.New("must provide task ID.")
+	}
+	err := cli.Logs(os.Stdout, map[string]string{"task": args[0]})
```

Comment:
Should we setup a signal handler here to cancel the context and cleanly as possible exit?

Reply:
yup. fixed.